### PR TITLE
Remove signup resolvers

### LINKED ISSUE #275: Remove createShiftSignup and updateShiftSignup
TO DO: remove createShiftSignup and updateShiftSignup because we should now be using `upsertDeleteShiftSignups`

_Originally posted by @LenaNguyen in https://github.com/uwblueprint/sistering/pull/253#pullrequestreview-948873157_

### PR DESCRIPTION
**Pull request by Boyazhang1**
## Ticket link

Closes #275

## Implementation description
* Removes all implementations of createShiftSignup and updateShiftSignup in codebase 
* replaces use of updateshiftsignup with upsertDeleteShiftSignups in RemoveVolunteerModal.js

## Steps to test
1. Navigate to http://localhost:3000/admin/schedule/posting/1
2. Click on the shift from 1:30-2:30pm on July 27th
3. Beside available volunteers in the side panel, click "edit", select valorie volunteer and click save
4. Click review, and remove Valerie volunteer from Wednesday, July 27th 
5. Click back to editing, and verify Valerie volunteer is properly removed
Screen Shot 2022-07-02 at 6 51 18 PM

## What should reviewers focus on?
* all implementations of createShiftSignup and updateShiftSignup are removed without creating new bugs

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Review by MatoPlus — COMMENTED** [review]
Looks like there's a lot of linting errors on both frontend and backend folders. Can you run the `yarn fix`s?

File: backend/middlewares/auth.ts [hunk 1]
```diff
@@ -81,98 +81,6 @@ export const isAuthorizedByUserId = (userIdField: string) => {
...
-    if (!authorized) {
-      throw new AuthenticationError(
-        "Failed authentication and/or authorization by userId",
-      );
-    }
-
-    return resolve(parent, args, context, info);
-  };
-};
-
```
Comment (MatoPlus) [inline backend/middlewares/auth.ts:175]: hmm are these not used anymore? I thought many the upsertDeleteShifts resolver would use one of these middleware to some degree

**Review by MatoPlus — COMMENTED** [review]
Aside from the little comment I made to investigate later, changes looks good to me! I'll leave it to @neelismail01 to do the final test/stamp

File: backend/middlewares/auth.ts (hunk 1 above)
Comment (Boyazhang1) [reply backend/middlewares/auth.ts:175]: I remember they showed up as grey in the imports, but I will double check as well 😃

**Review by neelismail01 — COMMENTED** [review]
I just tested your PR and it seems that Valerie volunteer isn't removed on the side panel when you remove the user in the Review page. Was this working for you locally?

**Review by neelismail01 — APPROVED** [review]

File: backend/middlewares/auth.ts (hunk 1 above)
Comment (MatoPlus) [reply backend/middlewares/auth.ts:175]: okay yea I don't think we need this anymore. Changes looks good!

**Pull request merged**